content/paths/folder_lock__get_folder_locks.yml

```
@@ -0,0 +1,61 @@
+---
+operationId: get_folder_locks
+
+summary: List folder lock information
```

```suggestion
summary: List folder locks on folder
```

I've generally been keeping the word "information" out of titles cause it feels.... redundant? Does this only ever return one lock?

---

+1, noted. This endpoint is set up to be able to return multiple locks per folder.